packages/image_picker/image_picker_ios/ios/image_picker_ios/Sources/image_picker_ios/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -128,64 +128,20 @@ - (void)start {
 - (void)processImage:(NSData *)pickerImageData API_AVAILABLE(ios(14)) {
   UIImage *localImage = [[UIImage alloc] initWithData:pickerImageData];
 
-  PHAsset *originalAsset;
-  // Only if requested, fetch the full "PHAsset" metadata, which requires  "Photo Library Usage"
-  // permissions.
-  if (self.requestFullMetadata) {
```

Comment:
Did `PHPickerViewController` always return the full metadata after iOS 14? Curious why have separate branches here in the first place.

Reply:
It seems that `PHPickerViewController` returns full metadata. The user has the option to remove certain metadata such as location 
![Simulator Screenshot - iPhone 16 - 2024-12-27 at 09 42 27](https://github.com/user-attachments/assets/259a0789-733e-439f-8cc4-b227af0fc8e0)
but by default they are included. 
![Simulator Screenshot - iPhone 16 - 2024-12-27 at 09 42 22](https://github.com/user-attachments/assets/0dc0fa49-d63c-452d-9f2f-b8457c621140)